docs/guides/update-guide/elasticsearch/7-to-8.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 7-to-8
+title: Update Elasticsearch 7 to 8
+description: "Review which adjustments must be made to migrate from Elasticsearch 7 to Elasticsearch 8."
+---
+
+The Camunda Platform is not compatible with the Elasticsearch 8 Curator. This Curator is used to manage
+[data retention](../../../components/concepts/data-retention.md) within Elasticsearch. To upgrade to Elasticsearch 8,
```

Comment:
```suggestion
[data retention](../../../components/concepts/data-retention.md) within Elasticsearch. To update to Elasticsearch 8,
```

Reply:
💡 We could use https://vale.sh/ to lint our docs for such words and instead of failing the CI we could use it to warn the author and reviewers of the usage with a note about when to use which form.

Reply:
Nice catch @korthout! I love this idea.